lib/worker.js

```diff
@@ -139,6 +139,10 @@ class Worker extends BaseService {
                 basePath = this._basePath;
             }
             service.conf.worker_id = this.config.worker_id;
+            service.conf.parent = {
+                logging: this.config.logging,
+                metrics: this.config.metrics,
+            };
```

Comment:
WE already expose the config to the user, maybe it's better to add these properties into the config?

Reply:
I'm not sure what you mean.  Isn't that what this is doing?

Reply:
Oh, indeed, sorry. 

Not sure about the `parent` though. Do you think it's needed to wrap them into `parent`?

Reply:
There's already a key at the top level called `metrics`.  I'm not married to `parent` though, suggestions welcome.